src/components/Message/Message.stories.tsx

```diff
@@ -0,0 +1,173 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+import { useState } from 'react'
+
+import { Hierarchy, Size } from '../Button/Button.types'
+import { Button } from '../Button'
+import { FeedbackMessagePositionType } from '../../hooks/useFeedbackMessage/useFeedbackMessage.types'
+import { Icon } from '../Icon'
+import { Message } from '.'
+import { useFeedbackMessage } from '../../hooks/useFeedbackMessage/useFeedbackMessage'
+
+const meta = {
+  args: {
+    message: 'This is a Feedback Message',
+  },
+  component: Message,
+} satisfies Meta<typeof Message>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const FeedbackMessages: Story = {
+  decorators: [(_Story, context) => {
+    const { info, loading, success, warning, error } = useFeedbackMessage()
+
+    const { message } = context.args
+
+    return (
+      <div style={{ display: 'flex', gap: '4px' }}>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="blue" name="PiInfo" size={16} />}
+          onClick={() => info({ message })}
+        >
+Info
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="green" name="PiCheck" size={16} />}
+          onClick={() => success({ message })}
+        >
+Success
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon name="PiSpinner" size={16} />}
+          onClick={() => loading({ message })}
+        >
+Loading
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="orange" name="PiWarning" size={16} />}
+          onClick={() => warning({ message })}
+        >
+Warning
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="red" name="PiXCircle" size={16} />}
+          onClick={() => error({ message })}
+        >
+Error
+        </Button>
+      </div >
+    )
+  }],
+}
+
+export const FeedbackMessageWithExtraContent: Story = {
+  decorators: [(_Story, context) => {
+    const { message } = context.args
+
+    const { success, dismiss } = useFeedbackMessage()
+
+    const onDismiss = (): void => { dismiss('messageKey') }
+
+    const onClick = (): void => {
+      success({
+        extra: <Button size={Size.Small} onClick={onDismiss}>Dismiss</Button>,
+        key: 'messageKey',
+        message,
+        sticky: true,
+      })
+    }
+
+    // TODO: Include a theme here
```

Comment:
The problem here is that the component rendered inside the feedback message does not include the Theme since it is generated by the hook and it's not coming from the `Story` parameter. Therefore, it does appear with a different blue. I'd remove the comment for now, and I'll verify this issue won't happen when importing the library in other applications.